### PR TITLE
climatology bias

### PR DESCRIPTION
**Pull request by AliciaTak**

### CODE REVIEW

**Review by DamienIrving — COMMENTED** [review]
In the seasonal cycle notebooks I've basically created a metric that combines all 12 monthly climatology bias maps into one value (the sum of the absolute value of the 12 monthly bias values). 

I wonder if the code in these climatology bias notebooks that plots the map for eacah month should be added to the seasonal cycle notebooks as an option? (i.e. we wouldn't plot all the maps for every task/RCM but maybe just for a key one or two).

In other words, the seasonal cycle and climatology bias notebooks kind of duplicate each other and should probably be combined.

It might be easier to merge this PR and do the combining later (I'd be happy to do it if the seasonal cycle notebooks are a bit confusing to understand).

**Comment by AliciaTak** [comment]
> In the seasonal cycle notebooks I've basically created a metric that combines all 12 monthly climatology bias maps into one value (the sum of the absolute value of the 12 monthly bias values).
> 
> I wonder if the code in these climatology bias notebooks that plots the map for eacah month should be added to the seasonal cycle notebooks as an option? (i.e. we wouldn't plot all the maps for every task/RCM but maybe just for a key one or two).
> 
> In other words, the seasonal cycle and climatology bias notebooks kind of duplicate each other and should probably be combined.
> 
> It might be easier to merge this PR and do the combining later (I'd be happy to do it if the seasonal cycle notebooks are a bit confusing to understand).

@DamienIrving, I understand your point. It would be good to add the precipitation ones, maybe fewer of the temperature ones. One notebook is around ~17MB; would that fit in the seasonal cycle one? The limit is 49MB, I think. I leave this to you to reorganise :) Thanks!

**Review by DamienIrving — COMMENTED** [review]
These new extreme event and drought intensity notebooks look good. Few minor comments:
- The title for the extreme event plots say "annual mean" in them, which I'm assuming is a hangover from another notebook that you copied to start work on this one. Perhaps the title should say something like: "Bias in 1-in-10 year extreme high precipitation over the 1980-2019 period (model minus obs)"
- I think we might need to make the MAE text larger in all our plots - it can be so small that it's hard to read
- You don't need the words "drought magnitude" in the subplot headings because it's already in the suptitle at the top

**Review by DamienIrving — COMMENTED** [review]
Very minor comment (sorry!) - it might be worth adding row headings to the drought intensity plots (something like "1yr" "2yr" "5yr") to show what the window is for each row of plots. https://stackoverflow.com/questions/25812255/row-and-column-headers-in-matplotlibs-subplots

**Comment by AliciaTak** [comment]
> Very minor comment (sorry!) - it might be worth adding row headings to the drought intensity plots (something like "1yr" "2yr" "5yr") to show what the window is for each row of plots. https://stackoverflow.com/questions/25812255/row-and-column-headers-in-matplotlibs-subplots

Oh, sorry for that! Happens when I work on multiple notebooks simultaneously. I'll update it.

**Comment by AliciaTak** [comment]
> > Very minor comment (sorry!) - it might be worth adding row headings to the drought intensity plots (something like "1yr" "2yr" "5yr") to show what the window is for each row of plots. https://stackoverflow.com/questions/25812255/row-and-column-headers-in-matplotlibs-subplots
> 
> Oh, sorry for that! Happens when I work on multiple notebooks simultaneously. I'll update it.

@DamienIrving are the headings "annual", "2-yr moving window", and "5-yr moving window" ok? Or do you have another suggestion for the "annual" one?

**Comment by DamienIrving** [comment]
Those seem fine. If you need a shorter label "2yr window" or even just "2yr" would work, but "2yr moving window" is clearest if you've got the space.

**Review by DamienIrving — APPROVED** [review]
Looks good, @AliciaTak. Feel free to merge if/when you're ready.
